data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1875 @@ mission "Stone of our Fathers 5"
+			choice
+				`	"Beat it, lady."`
+					goto lady
+				`	"Thanks, but I am just passing through."`
+					goto stars
+				`	"My home is among the stars."`
+					goto stars
+			label lady
+			`	A frown flickers across Carla's face for a brief second before her professional smile reasserts itself. "Trust me, why rent when you can own? Why move a large family to an empty world with poor restaurant selections, when you could move them here?".` 
+			`	She stands and smooths her suit jacket. "When you change your mind, I'll be here."`
+			`	With that she's gone and you are free to turn your attention back to the God-Stone.`
+					goto elfstone
+			label stars
+			`	Carla rewards you with a condescending smile. "Ah, darling. You might work in space, but eventually, you'll need a world to call home."`
+			`	She stands and smooths her suit jacket. "When you are ready to make the real-estate leap, you know where to find me."`
+			`	She winks at you and then strolls off in the crowds of shoppers. Alone, you turn your attention back to the God-Stone.`
+					goto elfstone
+			label elfstone
+			`	The parts of the God-Stone that are exposed at this level are dull and lusterless. You search for a pattern, but can make nothing out due to how little of the wall is visible. It's sectioned off with every shop and restaurant having its own piece. As you stare at it you feel nothing, just an emptiness. You do notice a plaque by a large part of unobstructed wall.`
+			choice
+				`	(Read the Plaque.)`
+					goto plaque
+				`	(Return to your ship.)`
+					goto ship
+			label plaque 
+			`	The plaque labels the God-Stone as 'Colonial Graffiti' and reads:`
+			`	During the Colonization of The Deep in the 26th Century, the earliest colonists  erected large metal slabs and wrote nonsensical pseudo-letters on them. It is believed that this was done in order to raise the prestige of their early worlds. They would claim that they were powerful alien artifacts, or that they were made by an earlier population of humans who had arrived on the world thanks to the intercession of fairies.`
+			`	Wanting to profit from superstitious nonsense, many local communities promoted these outrageous ideas about the origins of these structures. Scientists have determined that these structures were once large segments of the metal hulls of the original colony ships. Although, to date, there has been some trouble identifying exactly what kind of metal they were made from.`
+			`	Most worlds in The Deep have 'Colonial Graffiti', and they are proud reminders of our enterprising colonial origins. They are sometimes known in older texts, and by some outdated communities as God-Stones and Elf-Stones.`
+			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 
```

Comment:
My opinion here is there has to be an 'official' story.

 The current Deep missions have the Elves bringing humans to The Deep story as somewhat of a reveal. That means it isn't common knowledge, or if it is, it is treated as a folk-legend and not a serious one. In which case there has to be a serious official story. 

So, what we're seeing here is not any real departure from what is already established by the lore, we're just seeing how the 'official story' rationalizes elements of the 'fairy-tale'.

Reply:
Right, but this is not just the official story rationalising it. Particularly with the commentary from the shopping centre, the presentation in this context is actively denigrating it. There is a difference.

When there's something you don't want people to look at too closely, you try and avoid drawing attention to it. Effectively shouting, "This is not important!" from the rooftops doesn't do that.

Obviously the official line says, "We likely came here in the 26th Century, and the rest are just folktales."

But to draw from your example, this particular sign would be like putting up a sign in a deeply conservative christian area that says, "The bible is a silly book written by a lunatic 2000 years ago. This is why we've taken this cross and hung advertisements from it! Obviously using the cross for profit-making by non-christian businesses is still respectfully including our past in our current community!"
Can you imagine the outcry, the protests, and the vandalism that would occur in that case today?

It's wildly, and unnecessarily, hostile towards the narrative.

I would actually expect a very long plaque that spends like 1000 words talking about the mythos, and how it waned in significance on Asgard, and what the official line is, and what efforts were made to try and prove the mythos before ultimately failing to find evidence, etc. etc. 
Like, if there's a plaque should be boring. It should work to actively deter people from putting in the effort to care, and be politically neutral enough to be only of interest to people who enjoy the fairytale for its own sake.

Reply:
I would say the best way to dissuade a narrative would be to denigrate it. _Only the loonies, REALLY, believe that we came from elves_. 

Also, this is also plaque in a very modern, very materialistic area, this is communicated to the pilot in multiple ways, and one of them is the plaque itself. Hence, why this is not the same as "deeply conservative Christian area", but the reverse. 
It could be 1000 words and more nuanced, but would certainly be a drag to read, and it also lessens the weight and purpose of the plaque. The value of the plaque is:

- It communicates to the player the "official story" regarding the God-Stones. 'Colonial Grafiti' goes hand-in-hand with the previous established lore on how the Deep handle the mystery of their origins.
- It communicates to the player what the prevalent attitude of this community is towards 'traditional' ideas. That they are, essentially, non-believers. 

An important theme in this mission is the tension between the old ways; the rural lower-class traditional people, and a more modern, more elite, more universal culture. A kind of tension like this exists everywhere, and certainly would exist in a nation that is actively trying to keep its populace from knowing the secret of its past, a 'secret' that traditional communities might still have access too.

I think you aren't considering the ramifications of what we learn in the "Deep: Questions mission" line. 
An archaeologist, with one piece of wood from one old building was able to find credible evidence that The Deep civilization pre-dates spaceflight by quite a bit. This means that any number of Deep researchers would have been able to easily discover the same. Which means its not a fairy-tale, its easily established fact for anyone who wants to do the science. Which means that in order for that information to be kept secret, The Deep has to be actively suppressing it. They have over 15 billion people on their worlds, and scientists would certainly be doing digs and dating things and curious about their origins, so just a passive" dissuading people from looking", is not nearly enough to cover that. Furthermore, as is demonstrated in the Deep: Questions mission, they actively restrict access to their region to outsiders, like the archaeologist, who could test the matter. Which means they either are in control of all their academics, or they are suppressing all their academics. Therefore every museum, every textbook, every research paper published, would out of necessity all support the lie. Otherwise, none of this would news to the archaeologist in that mission.
Which means every Deep child grows up learning the lie, and maybe their grandmothers tell them stories about elves and they like the elf-stories, they still believe that at its core its a bunch of nonsense.

Reply:
You say `just a passive" dissuading people from looking", is not nearly enough to cover that.` but there are ways to intercept people who are looking, who are seeking grants for research etc. etc.

The problem as you so rightly point out is that the evidence actually is there to prove it really quite easily. Which means the most important feature in controlling the information, is _reducing the number of people who are willing to look._

Everything official needs to support the lie, I agree.
The problem is that, everything that supports the lie needs to do so in such a way as to not actively try and stoke the "loonies" into continuously trying to prove it wrong. If almost anyone can find proof, then the last thing you want to do is harass the "loonies" by actively denigrating them, because you'll have complete randoms coming out of the woodwork every two seconds with more proof. All just because you pointed your finger and said "no you can't". People are stubborn, if you attack them they'll fight back.

The more often people stick their head up and you shut them down, the greater the number of people who are going to pay attention, maybe look into it themselves. This is a problem if there's actual substance to it.

This is why in modern politics, the most popular way to escape a scandal is not to fight it, but to announce something else controversial and move on. You simply keep as much attention off of it as possible.

`Hence, why this is not the same as "deeply conservative Christian area", but the reverse.`
This is a wildly public location near a major spaceport, people who are not part of the modern elite would encounter it frequently enough for it to be an issue, and there's nothing quite like an outrage to motivate traditionalists with nothing better to do into organising a road trip for a large group all to stage a protest.

The best suppression is a message that says, "We don't believe, but we don't care if you do, because we have different things to care about." Because that encourages the people who do believe to just stay out of the way, and live their lives in peace, where they can't or won't encourage anyone to do their own research.

Active hostility to the message gives credence to it, the absolute last thing you want to do when suppressing a narrative is give credence to something that actually is credible.

Reply:
`It could be 1000 words and more nuanced, but would certainly be a drag to read,`
Also, for the record, I'm not suggesting you actually write all that out, I'm suggesting you summarily state that that is the content of plaque and move on. Implicitly suggesting that the full contents are below the notice of the vast majority of people.

Reply:
I have edited the plaque description to make it less condescending.